style.css

```diff
@@ -2,650 +2,388 @@
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+#contact {
+  width: 59px;
+  height: 20px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+#normal_btn {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 142px;
+  top: calc(50% - 40px / 2);
+  border-radius: 8px;
+}
+
+#icons {
+  width: 24px;
+  height: 24px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+#union1 {
+  position: absolute;
+  width: 22px;
+  height: 18px;
+  left: 1px;
+  top: 3px;
+  background: #3e3c3c;
+}
+
+#vectorx {
+  position: absolute;
+  left: 25%;
+  right: 25%;
+  top: 33.33%;
+  bottom: 50%;
+  border: 2px solid #67798e;
+}
+
+#rec {
+  box-sizing: border-box;
+  position: absolute;
+  left: 4.17%;
+  right: 4.17%;
+  top: 12.5%;
+  bottom: 12.5%;
+  border: 2px solid #67798e;
+  border-radius: 4px;
+}
+
+#social_icons {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  width: 20px;
+  height: 98.96px;
+  left: 36px;
+  top: 394px;
+}
+
+#twitter {
+  flex: 1;
+}
+
+#linkedin {
+  flex: 1;
+}
+
+#medium {
+  flex: 1;
+}
+
+#angel_list {
+  flex: 1;
+}
+
+#github {
+  flex: 1;
+}
+
+#headline {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1px;
+  gap: 24px;
+  position: absolute;
+  width: 566px;
+  height: 739px;
+  left: 143px;
+  top: 72px;
+}
+
+#primary_text {
+  width: 566px;
+  height: 160px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 82px;
+  color: #f55800;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+#supporting_text {
+  width: 566px;
+  height: 24px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+#description {
+  width: 566px;
+  height: 123px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  color: #f4f5f7;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+#base1 {
+  position: absolute;
+  width: 1440px;
+  height: 870px;
+  z-index: -1;
+  background: #1c1a19;
+}
+
+#illustration {
+  position: absolute;
+  left: 96.37%;
+  right: -96.23%;
+  top: 0%;
+  bottom: 0.1%;
+  transform: matrix(-0.96, 0.27, 0.27, 0.96, 0, 0);
+}
+
+#illustration_header {
+  position: absolute;
+  width: 1003.13px;
+  height: 769.27px;
+  left: 1744.68px;
+  top: -135px;
+  transform: matrix(-0.96, 0.27, 0.27, 0.96, 0, 0);
 }
```

Comment:
- [ ] As stated the project requirement You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly work on it just like the Figma design.

The Figma design for mobile screen size at 375px Look likes this : 
 
![Capture](https://user-images.githubusercontent.com/93598189/205016136-a3d0b42d-dc67-4583-b40d-c7663525e177.PNG)


Your Project Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/205016525-ceabdb14-9db5-4b4d-9a10-e1fcd61a7abb.PNG)

Reply:
This was broken while I was fixing linters, fixed now.